apps/fz_http/test/support/test_helpers.ex

```diff
@@ -16,16 +15,25 @@ defmodule FzHttp.TestHelpers do
     UsersFixtures
   }
 
-  def restore_env(key, val, cb), do: restore_env(:fz_http, key, val, cb)
+  import Mox
 
-  def restore_env(app, key, val, cb) do
-    old = Application.fetch_env!(app, key)
-    Application.put_env(app, key, val)
-    Cache.put!(key, val)
+  def stub_conf(key, val) do
+    stub(Cache.Mock, :get!, fn conf_key ->
```

Comment:
You can easily use the process dictionary here by overriding configuration per test process, without the use of Mox (runtime swapping of code or introducing behaviors and module implementations): https://gist.github.com/AndrewDryga/b6e1e3662181afe709b575338da0cd6c